src/bin/ambit/main.rs

```diff
@@ -146,10 +151,10 @@ fn run() -> AmbitResult<()> {
         cmd::init(force)?;
     } else if let Some(matches) = matches.subcommand_matches("clone") {
         let force = matches.is_present("force");
-        let origin = matches.value_of("ORIGIN").unwrap_or("");
-        cmd::clone(force, origin)?;
+        let git_arguments = matches.values_of("GIT_ARGUMENTS").unwrap().collect();
+        cmd::clone(force, git_arguments)?;
     } else if let Some(matches) = matches.subcommand_matches("git") {
-        let git_arguments: Vec<_> = matches.values_of("GIT_ARGUMENTS").unwrap().collect();
+        let git_arguments = matches.values_of("GIT_ARGUMENTS").unwrap().collect();
```

Comment:
Fetching `git_arguments` is repetitive here. Perhaps there is a more eloquent way of accomplishing this to avoid repetition?